test/functional/feature_dusd_loans.py

```diff
@@ -535,30 +535,21 @@ def check_looped_dusd(self):
                                 ERR_STRING_MIN_COLLATERAL_DFI_PCT,
                                 self.takeloan_withdraw, vault_id, "1.00000000@DUSD", 'takeloan')
 
-        self.nodes[0].setgov({"ATTRIBUTES": {'v0/params/feature/allow-dusd-loops': 'true'}})
-        self.nodes[0].generate(1)
-
-        #still not allowed before hardfork
-        assert_raises_rpc_error(-32600,
-                                ERR_STRING_MIN_COLLATERAL_DFI_PCT,
-                                self.takeloan_withdraw, vault_id, "1.00000000@DUSD", 'takeloan')
-        self.goto_next_height()
-
         # min coll ratio still applies
         assert_raises_rpc_error(-32600,
                                 "Vault does not have enough collateralization ratio defined by loan scheme - 148 < 150",
                                 self.takeloan_withdraw, vault_id, "67.00000000@DUSD", 'takeloan')
 
-
+        self.goto_next_height()
+        self.nodes[0].setgov({"ATTRIBUTES": {'v0/vaults/dusd-vault/dusd-vault-enabled': 'true'}})
```

Comment:
Either `v0/vaults/dusd-vault` as bool, or `v0/vaults/dusd-vault/enabled` if we have to accommodate for future unknowns.

Reply:
Resolved in commit 2496058